collector-exoplayer/src/main/java/com/bitmovin/analytics/exoplayer/ExoPlayerAdapter.java

```diff
@@ -104,6 +122,7 @@ public long getPosition() {
     @Override
     public void onTimelineChanged(Timeline timeline, Object manifest, int reason) {
         Log.d(TAG, "onTimelineChanged");
+        updateLoadedModules();
```

Comment:
would it make sense to first initialize the loaded modules in the constructor?

Reply:
Modules can be registered/unregistered anytime. 
When the `createMediaSource` is called, the respective module (HLS, DASH) is registered. At this point in time, the collector is already instantiated. So I was looking for an event that would be triggered when the manifest changes (here we can say for sure that the module was loaded because it is required). The `onTimelineChanged` event is called when the manifest changes.